db/mongodb_test.go

```diff
@@ -76,4 +78,8 @@ func TestMongoDB(t *testing.T) {
 	if metadata2 != "forty-two" {
 		t.Errorf("expected foruty-two as the answer, got %s", metadata2)
 	}
+	if err := transform.ValidateIndexes([]string{"teste.index1"}); err == nil {
```

Comment:
A gente não precisa testar o `ValidateIndexes` quando implementamos o `ExtraIndexes`. Quando implementamos o `ExtraIndexes` precisamos testar o `ExtraIndexes`

```suggestion
	if err := db.ExtraIndexes([]string{"teste.index1"}); err == nil {
```

Reply:
Verdade! Até pq nele já tem o validate!!!